includes/class-sm-post-types.php

```diff
@@ -212,29 +214,29 @@ public static function register_post_types() {
 
 		register_post_type( 'wpfc_sermon', apply_filters( 'sm_register_post_type_wpfc_sermon', array(
 			'labels'              => array(
-				'name'                  => __( 'Sermons', 'sermon-manager-for-wordpress' ),
```

Comment:
This definitely has to be reverted. I know that it's a pain in the butt to write out the domain each time - but WordPress' GlotPress depends on the hardcoded string. :disappointed:

Reply:
Yeah, I thought that might be the case (hence why that change is in it's own commit). 

What I am curious about though is why does glotpress need that to be a hard-coded string. Is there not a WordPress API for that or something? Also, why do we need to support glotpress?

Reply:
>What I am curious about though is why does glotpress need that to be a hard-coded string.

Tbh, I'm not sure - it's the way how it's coded, I guess. The domain should (must?) be the same as plugin slug on wp.org for it to work. 

>Also, why do we need to support glotpress?

It's the interface for easily translating Sermon Manager to other languages: https://translate.wordpress.org/projects/wp-plugins/sermon-manager-for-wordpress

Reply:
Gotcha. Reverted.